components/canvas/WaitingNode.tsx

```diff
@@ -137,7 +138,10 @@ export const WaitingNode = ({
         highlighted={isDropTarget && isValidDropTarget}
         darkened={isValidDropTarget === false}
       >
-        <NodeDescription header={type} />
+        <NodeDescription
+          header={!description ? type : undefined}
```

Comment:
Since description is of type `string`, is there a need for a ternary operator?

Reply:
If the description is empty we pass type to the component so that we can render the type as text. Otherwise we pass in undefined so that we can avoid rendering the Typography component that would normally contain the type text. Do you have another suggestion?